src/Sarif/Autogenerated/FileChange.cs

```diff
@@ -34,11 +34,11 @@ public SarifNodeKind SarifNodeKind
         /// <summary>
         /// The location of the file to change.
         /// </summary>
-        [DataMember(Name = "fileLocation", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "fileLocation", IsRequired = true)]
         public FileLocation FileLocation { get; set; }
 
         /// <summary>
-        /// An array of replacement objects, each of which represents the replacement of a single range of bytes in a single file specified by 'fileLocation'.
+        /// An array of replacement objects, each of which represents the replacement of a single range of bytes in a single file specified by 'uri'.
```

Comment:
>range of bytes [](start = 98, length = 14)

"range of bytes" => "region". Let's keep track of the changes I mention here and push them back into OASIS.

Reply:
fixed. note that i will push the updated SDK schema back to OASIS in a separate change. the goal is a shared file.

---
In reply to: [214740638](https://github.com/Microsoft/sarif-sdk/pull/1067#discussion_r214740638) [](ancestors = 214740638)